src/organisms/LocationsPage.jsx

```diff
@@ -82,13 +91,43 @@ function renderLocations(locations) {
   );
 }
 
-const LocationsPage = () => {
+const LocationsPage = ({ location }) => {
```

Comment:
This will most likely trigger CodeClimate to say the function is too long, but I think it is valid and does it's job effectively.